src/picongpu/include/particles/pusher/particlePusherAxl.hpp

```diff
@@ -37,7 +37,11 @@ namespace picongpu
         template<class Velocity, class Gamma>
         struct Push
         {
-
+            /* this is an optional extention for sub-sampling pushes that enables grid to particle interpolation
```

Comment:
nevertheless I personally would add an additional abstraction layer because I can use any pusher (simple momentum-advancing algorithm) with sub-stepping.